diff -up hplip-3.9.8/prnt/drv/hpcups.drv.in.requirespageregion hplip-3.9.8/prnt/drv/hpcups.drv.in
--- hplip-3.9.8/prnt/drv/hpcups.drv.in.requirespageregion	2009-08-28 08:59:11.086208461 +0100
+++ hplip-3.9.8/prnt/drv/hpcups.drv.in	2009-08-28 09:35:30.095904252 +0100
@@ -111,6 +111,8 @@ Throughput 1
 Filter "application/vnd.cups-raster 0 hpcups"
 //Filter "application/vnd.cups-command 0 commandtohpijs"
 
+Attribute "RequiresPageRegion" "All" "True"
+
 {
 // Manufacturer - PPD spec has "HP" registered for "Hewlett-Packard"...
 Manufacturer "HP"
